.github/workflows/dotnet_format.yml

```
@@ -0,0 +1,48 @@
+      - uses: actions/setup-dotnet@v4
+        with:
+          global-json-file: global.json
+      - uses: actions/cache@v4
+        with:
+          path: ~/.nuget/packages
+          # Look to see if there is a cache hit for the corresponding requirements file
+          key: ${{ runner.os }}-nuget-${{ hashFiles('Directory.Packages.props') }}
+          restore-keys: |
+            ${{ runner.os }}-nuget
+      - name: Run clean code
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:6:101: Double quote to prevent globbing and word splitting

(shellcheck)